android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1099,6 +1128,45 @@ private WritableArray getVideoTrackInfo(int trackRendererIndex) {
         return videoTracks;
     }
 
+    private int getScreenShortestSide(ThemedReactContext context) {
+        if (context == null) {
+            // No context so we fallback to max int
+            return (int)Integer.MAX_INT;
+        }
+        Display display = context.getCurrentActivity().getWindowManager().getDefaultDisplay();
+        int realWidth;
+        int realHeight;
+
+        if (Build.VERSION.SDK_INT >= 17){
+            //new pleasant way to get real metrics
+            DisplayMetrics realMetrics = new DisplayMetrics();
+            display.getRealMetrics(realMetrics);
+            realWidth = realMetrics.widthPixels;
+            realHeight = realMetrics.heightPixels;
+
+        } else if (Build.VERSION.SDK_INT >= 14) {
+            //Reflection for this weird in-between time
+            try {
+                Method mGetRawH = Display.class.getMethod("getRawHeight");
```

Comment:
is it safe without proguard rules?

Reply:
oh, it seems to be handled automatically. [proguard docs](https://www.guardsquare.com/manual/home#reflection)